yarn.lock

```diff
@@ -1100,19 +1100,6 @@
   resolved "https://registry.yarnpkg.com/@base2/pretty-print-object/-/pretty-print-object-1.0.0.tgz#860ce718b0b73f4009e153541faff2cb6b85d047"
   integrity sha512-4Th98KlMHr5+JkxfcoDT//6vY8vM+iSPrLNpHhRyLx2CFYi8e2RfqPLdpbnpo0Q5lQC5hNB79yes07zb02fvCw==
 
-"@bbc/psammead-radio-schedule@5.1.30":
```

Comment:
Should this change be in this PR?

Reply:
I'm not sure what's happened but currently my other PRs have this change. This was the only internal dependency `@bbc/` package in the `yarn.lock` file. All other bbc/psammead package are not in here. I'm guessing `@bbc/psammead-radio-schedule@5.1.30` found it's way in somehow but is actually not supposed to be there.